### ISSUE TITLE
Errors with "Replace all code"

### ISSUE THREAD

**Issue by Tohkann** [sailro/Reflexil#66]
What is a step by step instruction on how to fix these problems? It happens when trying to 'compile', regardless if any new code is written or not. This is for a unity build. If other compiler profiles are chosen, the CS1703 error occurs. 

![errorcs0656](https://cloud.githubusercontent.com/assets/19458980/15382798/0ee6fe5e-1d53-11e6-979f-5ef1f091d022.jpg)

There were times in which these error messages didn't appear, but hitting the compile button did not seem to do anything, and so the "ok" button wasn't available to confirm any changes. But this situation, I cannot re-create.

**Comment by sailro** [comment]
Weird, is your assembly obfuscated ? 
Can you share with me this specific assembly ?

**Comment by Tohkann** [comment]
I am not sure I am allowed to share the assembly, as I am modifying an already existing one to include some code, all for personal use. I already know what to code to input, I just need the ability to actually inject it.

However I did verify the unmodified assembly and the following was the result
![verify_errors](https://cloud.githubusercontent.com/assets/19458980/15763947/d11ccfc2-28ec-11e6-8aa4-80b5ef7b53b8.jpg)

(A side note, I attempted to save the code out unmodified and rebuild it in Visual Studio, but got back many syntax errors like incorrect placement of '$' and '>' characters and missing ending brackets. So something is definitively up with the assembly)
